HelpSource/Reference/Key-Value-Pairs.schelp

```diff
@@ -44,7 +44,39 @@ CODE::
 [\freq -> 452, \amp -> 0.2].asDict
 (freq: 452, amp: 0.2).asDict
```

Comment:
This example fails for me, I think because `Dictionary:-asDict` should choose `IdentityDictionary` as its default class like `Collection:-asDict`.